types/cssnano/cssnano-tests.ts

```diff
@@ -0,0 +1,17 @@
+import * as cssnano from 'cssnano';
```

Comment:
It should be `export = require` here https://github.com/DefinitelyTyped/DefinitelyTyped#should-i-add-an-empty-namespace-to-a-package-that-doesnt-export-a-module-to-use-es6-style-imports